.github/workflows/ci.yml

```diff
@@ -21,28 +21,6 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0 ## Needed for Changesets to find `main` branch
-
-        ## https://github.com/changesets/changesets/issues/517
-      - name: Create git reference to main branch (for Changesets)
-        if: ${{ github.event.pull_request.title != 'Version Packages' }}
-        run: git pull --force --no-tags origin main:main
-
-      - uses: ./.github/actions/warm-up-repo
-
```

Comment:
This is still required for the CI as it pulls tools and runs `yarn install`

Reply:
Oops, didn't mean to remove this – restored